src/screens/ProfileScreen.js

```diff
@@ -57,6 +60,15 @@ export default class ProfileScreen extends React.Component {
     this.logout = this.logout.bind(this);
   }
 
+  componentDidMount() {
+    CategoryRequester.retrieveCategories().then((response) => {
```

Comment:
This seems really convoluted - when you pass it here (https://github.com/calblueprint/veterans360-mobile/pull/44/files#diff-e02136417cfd21210d7cd3c94f499ee9R278) the only place its being used is here (https://github.com/calblueprint/veterans360-mobile/blob/master/src/components/Resource.js#L71). It seems to me, if anything, this should be in the `Resource.js` and not passed as a prop. Not sure though @lbkchen